### PR TITLE
Update ps3 walk documentation with new inputs

### PR DESCRIPTION
**Pull request by AnExParrot**
https://github.com/NUbots/NUbots/pull/1236

### CODE REVIEW

File: src/book/02-system/02-subsystems/06-behaviour.mdx [hunk 1]
```diff
@@ -47,19 +55,30 @@ The [**fake/keyboardwalk** role](https://github.com/NUbots/NUbots/blob/main/role
 
 The [PS3 walk module](https://github.com/NUbots/NUbots/tree/main/module/behaviour/strategy/PS3Walk) is an interface for **testing** and **demonstrating** the walk and other functionalities using a **PS3 controller**.
 
-The module connects to the PS3 controller using bluetooth (or USB) and reacts based on joystick or button input.
+The module connects to the PS3 controller using USB and reacts based on joystick or button input.
```
Comment (ysims) [inline src/book/02-system/02-subsystems/06-behaviour.mdx:58]: Is there a problem wth bluetooth? We had it going early last year with the PS3 walk.

Comment (AnExParrot) [reply src/book/02-system/02-subsystems/06-behaviour.mdx:58]: It didn't work when we last tried it 2 weeks ago. I'll have another look on Friday.

**Review by LachlanCourt — DISMISSED** [review]
LGTM 🚀

**Comment by ysims** [comment]
Update the PR title to something that finishes the sentence, 'When merged, this PR will..."

**Comment by LachlanCourt** [comment]
Don't forget to run `yarn format` before pushing, that's failing the build atm

File: src/book/02-system/02-subsystems/06-behaviour.mdx [hunk 2]
```diff
@@ -66,15 +66,15 @@ The module connects to the PS3 controller using USB and reacts based on joystick
...
-| L2                        | Left kick.                                 |
-| DPAD Up                   | Perform Step Clap 1 move.                  |
-| DPAD Down                 | Perform Step Clap 2 move.                  |
-| DPAD Left                 | Perform Overhead Thrust Right move.        |
-| DPAD Right                | Perform Overhead Thrust Left move.         |
-| Triangle                  | Perform Star 1 move.                       |
-| Circle                    | Perform Star 2 move.                       |
-| Cross                     | Perform Crouch 1 move.                     |
-| Square                    | Perform Crouch 2 move.                     |
+| L2                        | Left  kick.                                |
```
Comment (ysims) [inline src/book/02-system/02-subsystems/06-behaviour.mdx]:
```suggestion
| L2                        | Left kick.                                |
```

**Review by Tom0Brien — APPROVED** [review]

**Pull request merged**